source/agora/node/admin/QRCodeInterface.d

```diff
@@ -0,0 +1,269 @@
+/*******************************************************************************
+
+    Provides QR service interface used by the node's web-based management
+    interface
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.node.admin.QRCodeInterface;
+
+import agora.common.Config;
+import agora.common.crypto.Key;
+import agora.common.Hash;
+import agora.common.Types;
+import agora.consensus.data.PreImageInfo;
+import agora.utils.Log;
+import agora.network.Clock;
```

Comment:
Why are you importing this Clock but then using the one from Phobos?

Reply:
If we want to write tests for this module, then we should use the Clock from the Network package. That way we can set the exact date and can even write tests. Obviously no tests were added to this PR, so it will have to be done later..

Reply:
I was going to use it at first. The change was big, so I left it for a while.
But now I've applied it.